pontoon/contributors/templates/contributors/verification_email.jinja

```diff
@@ -0,0 +1,8 @@
+Hey {{ display_name }},
+
+Please click on the link below within the next hour to verify your contact email address:
+{{ link }}
+
+You’re receiving this email, because you recently changed a contact email address on Pontoon. If this wasn’t you, please ignore this email.
```

Comment:
```suggestion
You’re receiving this email because you recently changed a contact email address on Pontoon. If this wasn’t you, please ignore this email.
```